internal/website/docs/support-channels.mdx

```diff
@@ -0,0 +1,20 @@
+---
+slug: support-channels
+title: Support Channels
+description: Discover the Faust Support Channels
+---
+
+## Need Faust support? Look no further!
+
+Use one of the channels below to contact us for support.
+
+<a href="/docs/img/telemetry-cli-prompt.png" target="_blank">
+  Github
+</a> - Faust Github documentation and codebase.
+
+<a href="/docs/img/telemetry-cli-prompt.png" target="_blank">
+  Discord
+</a> - Interactive chat support on Discord.
```

Comment:
```suggestion
<a href="https://github.com/wpengine/faustjs" target="_blank" rel="noopener noreferrer">
  GitHub
</a> - Faust Github documentation and codebase.

<a href="https://discord.gg/J2khkF9XYK" target="_blank" rel="noopener noreferrer">
  Discord
</a> - Interactive chat support on Discord.
```


- [More information on the target="_blank" vulnerability](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)

Reply:
Hi Matt, good work on adding the doc! It looks like you need a line break between the two links and discriptions, here, and to update the "last updated on" section:

![image](https://user-images.githubusercontent.com/50935135/193135461-ce8a1405-ee40-487b-80ed-3087799abc41.png)

I'm also not able to see the link for the doc in the sidebar- am I missing it somewhere?

![image](https://user-images.githubusercontent.com/50935135/193135872-711ecf98-a5da-4b79-a388-5fca5eec0879.png)

Reply:
After looking at the repo again, it seems like you need to add an object in the `sidebars.js` file in `internal/website`, with a type, label, and id for it to show up in the sidebar correctly.